packages/http/src/resources/__tests__/Stats.spec.ts

```diff
@@ -52,3 +61,16 @@ describe('get stats', () => {
     expect(stats.tokenSupply).toBe(33319737.03712976)
   })
 })
+
+describe('get counts', () => {
+  nock('https://api.helium.io').get('/v1/stats/counts').reply(200, {
+    data: countsFixture(),
+  })
+
+  it('retrieves network count stats', async () => {
+    const client = new Client()
+    const counts = await client.stats.counts()
+    expect(counts.blocks).toBe(805339)
```

Comment:
might want to check `.consensusGroups` too, just to double check that the camelCasing is working because it's the only two-word field

Reply:
added the `expect`, good shout